docs/cli.rst

```diff
@@ -62,60 +63,124 @@ primary difference with simple `user` command is that password is being imported
   docker-compose run --rm admin flask mailu user-import myuser example.net '$6$51ebe0cb9f1dab48effa2a0ad8660cb489b445936b9ffd812a0b8f46bca66dd549fea530ce' 'SHA512-CRYPT'
 
 user-delete
-------------
+-----------
 
 .. code-block:: bash
 
   docker-compose exec admin flask mailu user-delete foo@example.net
 
+config-dump
+-----------
+
+The purpose of this command is to dump domain-, relay-, alias- and user-configuration to a YAML template.
+If you want to export non-hashed secrets you have to add the ``--secrets`` option.
+Only non-default options are dumped. If you want to dump all options use ``--full``.
+
+.. code-block:: bash
+
+  docker-compose exec admin flask mailu config-dump > mail-config.yml
+
 config-update
 -------------
 
-The sole purpose of this command is for importing users/aliases in bulk and synchronizing DB entries with external YAML template:
+The purpose of this command is for importing domain-, relay-, alias- and user-configuration in bulk and synchronizing DB entries with an external YAML template.
 
 .. code-block:: bash
 
-  cat mail-config.yml | docker-compose exec -T admin flask mailu config-update --delete-objects
+  docker exec -i $(docker-compose ps -q admin) flask mailu config-update -nvd < mail-config.yml
 
-where mail-config.yml looks like:
+*(The current version of docker-compose exec does not pass stdin correctly, so you have to user docker exec)*
 
-.. code-block:: bash
+mail-config.yml looks like:
+
+.. code-block:: yaml
+ 
+  domains:
+    - name: example.com
+      alternatives:
+        - alternative.example.com
 
   users:
-    - localpart: foo
-      domain: example.com
+    - email: foo@example.com
       password_hash: klkjhumnzxcjkajahsdqweqqwr
       hash_scheme: MD5-CRYPT
 
   aliases:
-    - localpart: alias1
-      domain: example.com
+    - email: alias1@example.com
       destination: "user1@example.com,user2@example.com"
 
-without ``--delete-object`` option config-update will only add/update new values but will *not* remove any entries missing in provided YAML input.
+  relays:
+    - name: relay.example.com
+      comment: test
+      smtp: mx.example.com
 
-Users
------
+You can use ``--dry-run`` to test your YAML without omitting any changes to the database.
```

Comment:
small typo. Change 'omitting' to 'commiting'

Reply:
thanks for spotting this